config-overrides.js

```diff
@@ -87,6 +87,20 @@ function webpackOverride(config) {
   ])
 
   config.module.rules = modifyBabelLoaderRuleForBuild(config.module.rules)
+  config.module.rules.push({
+    test: /\.(js|jsx)$/,
+    exclude: /(node_modules|bower_components)/,
+    loader: 'babel-loader',
+    options: { presets: ['@babel/env', '@babel/preset-react'] }
+  })
+
+  config.module.rules.push({
+    test: /\.(m?js)$/,
```

Comment:
wanted to test locally if this caused any issues with webpack build because the build config was super wonky (we need to react-scripts eject). this is fine